_includes/checkout-v3-resource-models.md

```diff
@@ -515,6 +516,16 @@ Content-Type: application/json
 
 ## Paid
 
+The payment order response with status `Paid`, and the `Paid` resource expanded.
+Please note that the main code example is of a card payment. We have included
+`Paid` resources of the remaining instruments below the main code example.
```

Comment:
Since `paid` is referring to a property inside the JSON, we should use the same casing.

```suggestion
The payment order response with status `paid`, and the `paid` resource expanded.
Please note that the main code example is of a card payment. We have included
`paid` resources of the remaining instruments below the main code example.
```